src/CSSBattle/5_AcidRain/Tai.html

```diff
@@ -0,0 +1,72 @@
+<body>
+  <div class="rotated">
+    <div class="teardrop">
+      <div class="circle yellow"></div>
+    </div>
+    <div class="teardrop">
+      <div class="circle olive"></div>
+      <div class="square olive"></div>
```

Comment:
instead of circle and squares, consider using a single div and setting border-radius of all 4 sides differently